apps/backend/src/evaluations/dto/create-evaluation.dto.ts

```diff
@@ -1,26 +1,16 @@
 import {ICreateEvaluation} from '@heimdall/interfaces';
-import {
-  IsArray,
-  IsBoolean,
-  IsNotEmpty,
-  IsObject,
-  IsOptional,
-  IsString
-} from 'class-validator';
+import {IsArray, IsNotEmpty, IsOptional, IsString} from 'class-validator';
 import {CreateEvaluationTagDto} from '../../evaluation-tags/dto/create-evaluation-tag.dto';
 
 export class CreateEvaluationDto implements ICreateEvaluation {
   @IsNotEmpty()
   @IsString()
   readonly filename!: string;
 
-  @IsNotEmpty()
-  @IsObject()
   readonly data!: Record<string, any>;
```

Comment:
Is this field applicable at all to the CreateEvaluationDto at this point?

Reply:
Data is still needed as part of CreateEvaluationDto as the controller converts the blob to the data value on a new CreateEvaluationDto